tools/fv_eta.F90

```diff
@@ -778,7 +778,7 @@ subroutine var_les(km, ak, bk, ptop, ks, pint, s_rate)
   real ep, es, alpha, beta, gama
   real, parameter:: akap = 2./7.
 !---- Tunable parameters:
-  integer:: k_inc = 10   ! # of layers from bottom up to near const dz region
+  real:: k_inc = 10   ! # of layers from bottom up to near const dz region
```

Comment:
This should be `10.` not `10`

Reply:
While a semantic change, it will be cast correctly.  We can schedule it for a fix in master.